tools-ui/src/main/java/com/hedera/hashgraph/client/ui/CreatePaneController.java

```diff
@@ -1681,16 +1687,33 @@ private void storeOutputToBrowsedOutput(CreateTransactionType type) {
 		storeToOutput(type, fileService);
 	}
 
+	/**
+	 * Browse to a file and load the transaction
+	 *
+	 * @return a Tool transaction
+	 */
+	private File loadTransaction() {
+		logger.info("browsing transactions");
+
+		var file = BrowserUtilities.browseFiles(controller.getLastTransactionsDirectory(), createAnchorPane,
+				"Transaction", TRANSACTION_EXTENSION, SIGNED_TRANSACTION_EXTENSION, ZIP_EXTENSION);
+		if (file == null) {
+			return null;
+		}
+		controller.setLastBrowsedDirectory(file);
+		return file;
+	}
+
 	private boolean doNotStoreExpiringTransaction() {
 		var answer = true;
 		var date = getDate(datePicker, hourField, minuteField, secondsField, ZoneId.of(timeZone.getID()));
 		var now = new Date();
-		var millis = date.getTime() - now.getTime();
-		if (millis / 1000 < 120) {
+		var secs = date.getSeconds() - now.getTime() / 1000;
```

Comment:
Silly question, do we need to consider the possibility the time zones may be different?

Reply:
The `getDate` method will put everything into a standard Date, measured at UTC

Reply:
Make sense. Thx.